src/components/KeyboardNavigation/Provider.js

```diff
@@ -37,16 +37,28 @@ export default class KeyBoardProvider extends React.Component {
   static getDerivedStateFromProps(props, state) {
     if (!props.selected || props.selected.length === 0) return null;
 
+    const selectedValue = props.selected[0];
```

Comment:
I think `const selectedValue = props.selected[props.selected.length - 1];`. When the `isMultiSelect` prop equaling true is passed to `ItemSelectionProvider`, we concatenate a newly selected element to the `selected` array. See: https://github.com/ticketmaster/aurora/blob/master/src/components/List/helper.js#L10